src/shared/bpf-dlopen.c

```diff
@@ -51,8 +51,8 @@ int dlopen_bpf(void) {
                         DLSYM_ARG(bpf_link__fd),
                         DLSYM_ARG(bpf_map__fd),
                         DLSYM_ARG(bpf_map__name),
-                        DLSYM_ARG(bpf_create_map),
-                        DLSYM_ARG(bpf_map__resize),
+                        DLSYM_ARG(bpf_map_create),
```

Comment:
silly question: can this syntax support e.g. `bpf_map_create@LIBBPF_0.6.0` ? I don't see any '@' symbol in any DLSYM_* helper, but it'd be interesting for future proofing in theory.